process/project-lifecycle-documents/osv-schema_graduation_stage.md

```diff
@@ -0,0 +1,92 @@
+
+### Project has met all Incubating requirements
+  * n/a
+
+### List of project maintainers
+The project must have maintainers with a minimum of five different contributors from three different organizational affiliations.
+  * Oliver Chang, Google, @oliverchang
+  * Andrew Pollock, Google, @andrewpollock
+
+OSV Schema has had 62 contributors from 18 different organizations.
+
+Note: need to add a CONTRIBUTING.md doc to the project repo.
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be code needed to deliver OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * The mission of OSV is to develop a standard interchange format for describing vulnerabilities in open source packages.
+  * The OSV schema provides a human and machine readable data format to describe vulnerabilities in a way that precisely maps to open source package versions or commit hashes.
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+
+The OSV Schema is currently exported by:
+- [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+- [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
+- [Chainguard](https://packages.cgr.dev/chainguard/osv/all.json)
+- [Curl](https://curl.se/docs/vuln.json)
+- [GitHub Security Advisories](https://github.com/github/advisory-database)
+- [Global Security Database](https://github.com/cloudsecurityalliance/gsd-database)
+- [Go Vulnerability Database](https://github.com/golang/vulndb)
+- [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+- [LoopBack Advisory Database](https://github.com/loopbackio/security/tree/main/advisories)
+- [Malicious Packages Repository](https://github.com/ossf/malicious-packages)
+- [Mageia Advisories](https://advisories.mageia.org/)
+- [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
+- [OSV.dev maintained converters](https://github.com/google/osv.dev#current-data-sources) (Debian, Alpine, NVD)
+- [PyPI Advisory Database](https://github.com/pypa/advisory-database)
+- [Python Software Foundation Database](https://github.com/psf/advisory-database)
+- [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+- [Red Hat](https://security.access.redhat.com/data)
+- [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+- [Rust Advisory Database](https://github.com/RustSec/advisory-db)
+- [SUSE](https://www.suse.com/support/security/)
+- [Ubuntu](https://github.com/canonical/ubuntu-security-notices/)
+- [VMWare Photon OS](https://github.com/vmware/photon/wiki/Security-Advisories) (unofficial)
```

Comment:
```suggestion
```
I'm unable to validate this?

Reply:
the unofficial repo is here: https://github.com/captn3m0/photon-os-advisories/tree/main